### PR TITLE
Tests reorganization

### PR DESCRIPTION
**Pull request by lucatume**
Ticket: http://central.tri.be/issues/76364

This ticket overhauls the test files and structure in a way equivalent to the work done in https://github.com/moderntribe/events-pro/pull/586 by @bordoni

### CODE REVIEW

**Comment by lucatume** [comment]
This code is now ready for CR. All tests passing on Travis too
What does not work? The WPML integration.

File: tests.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
-# The Events Calendar tests
+# Events Calendar PRO tests
 
 This is a brief and quick guide that's covering the bare essentials needed to set up the tests on your local plugin copy.
 Please refer to [Codeception](http://codeception.com/docs) and [WP Browser](https://github.com/lucatume/wp-browser) documentation for any issue that's not TEC related.
 
 ## Set up
-After cloning the TEX repository on your local machine change directory to the plugin root folder and pull in any needed dependency using [Composer](https://getcomposer.org/):
+After clojning the TEX repository on your local machine change directory to the plugin root folder and pull in any needed dependency using [Composer](https://getcomposer.org/):
```
Comment (borkweb) [inline tests.md]: I'm excited to learn what `clojning` is!!!

Comment (lucatume) [inline tests.md]: It's a Tibetan tradition of monks trying to misspell any word they come across in a gesture of refusal of modern society tendency toward unachievable perfection in body and mind.

**Comment by borkweb** [comment]
Looks great! +1 for Travis pass 🗳

**Comment by lucatume** [comment]
Travis CI pass is my Pagespeed.

**Pull request merged**